code/datums/traits/negative.dm

```diff
@@ -424,6 +424,39 @@
 	to_chat(quirk_holder, "<span class='boldannounce'>Your [slot_string] has been replaced with a surplus prosthetic. It is fragile and will easily come apart under duress. Additionally, \
 	you need to use a welding tool and cables to repair it, instead of bruise packs and ointment.</span>")
 
+/datum/quirk/nugget
+	name = "Nugget"
+	desc = "An accident caused you to lose ALL of your limbs. There's no way your insurance payed for all those prosthetics!"
+	value = -1
+	var/slot_string = "limb"
+
+/datum/quirk/nugget/on_spawn()
+	var/mob/living/carbon/human/nuggeted = quirk_holder
+	for(var/X in nuggeted.bodyparts)
+		var/obj/item/bodypart/BP = X
+		if(BP.body_part != HEAD && BP.body_part != CHEST)
+			if(BP.dismemberable)
+				BP.dismember()
+	if(nuggeted.buckled)
+		nuggeted.buckled.unbuckle_mob(nuggeted)
+	nuggeted.suppress_bloodloss(1800) //stop them from bleeding out
+	nuggeted.update_body_parts(TRUE)
+
+/datum/quirk/nugget/post_add()
+	to_chat(quirk_holder, "<span class='boldannounce'>What cruel twist of fate has led to you arriving aboard a space station with no limbs?")
+
+/datum/quirk/corpse
+	name = "Corpse"
+	desc = "Something terrible happened on the shuttle to the station, you arrive dead as a doornail!"
+	value = -1
+
+/datum/quirk/corpse/post_add()
+	to_chat(quirk_holder, pick("<span class='boldannounce'>F", "<span class='boldannounce'>RIP", "<span class='boldannounce'>RIP in peace", "<span class='boldannounce'>RIP in pepperoni", "<span class='boldannounce'>You were THIS close to surviving"))
+	quirk_holder.adjustFireLoss(200)
+	quirk_holder.adjustBruteLoss(200)
+	quirk_holder.adjustToxLoss(200)
+	quirk_holder.adjustCloneLoss(200)
```

Comment:
How will they be revivable? I don't believe you can treat clone damage on a corpse and you can't defib at ~150 damage or so

Reply:
Fixed, now it's just a shitload of brute.